### PR TITLE
Update OpenBCI Logo in the GUI

### LINKED ISSUE #1010: Update OpenBCI Logo in the GUI
## Problem

The OpenBCI Logo has been updated on the official website.  https://www.openbci.com

## Expected 

We should use the new logo in the GUI in the TopNav, spinning loading gif, and on splash screen on app start.

Request access new logo assets from OpenBCI Design Team and try to resize/crop as needed. Try to keep the same dimensions as previous assets.

## Operating System and Version

All OS

## GUI Version

5.0.9

## Running standalone app

Standalone and Processing App

## Type of OpenBCI Board

N/A

## Are you using a WiFi Shield?

N/A

## Console Log

N/A

### PR DESCRIPTION
**Pull request by retiutut**
- [x] Fixes #1010 

- [x] Also found a null pointer error associated with Headplot and a threaded method "doHardCalcs".

- [ ] The rotating Cog gif needs to be looked at because it doesn't render entirely correct. Should be fixed or changed to something else before merging this PR.

### CODE REVIEW

**Comment by retiutut** [comment]
Screen Shot 2021-12-07 at 3 15 34 PM
)

**Comment by retiutut** [comment]
Screen Shot 2021-12-07 at 3 16 26 PM
)

**Review by Andrey1994 — CHANGES_REQUESTED** [review]
changes about processing 4 should not be in PR about logo change

File: OpenBCI_GUI/BoardBrainflow.pde [hunk 1]
```diff
@@ -16,6 +16,8 @@ abstract class BoardBrainFlow extends Board {
     protected int[] otherChannelsCache = null;
 
     protected boolean streaming = false;
+    protected double time_last_datapoint = -1.0;
```
Comment (Andrey1994) [inline OpenBCI_GUI/BoardBrainflow.pde:19]: and this too

**Pull request closed without merging**